SyncSettings.sublime-settings

```diff
@@ -1,4 +1,7 @@
 {
+    // Location for the .sync_settings configuration file
+    // Defaults: ~/.sync_settings
+    "config_location": "",
```

Comment:
this file is used for default values, please, try to add the default value here.

Reply:
Add this new property to `README.md` file.

Reply:
> this file is used for default values, please, try to add the default value here.

This will fall back to a default value. Since the default value is dependent on the system, we use a `""` for the setting. So `file_path = settings.get("config_location") or default_file_path` will choose `default_file_path` if the setting is ""

Reply:
okay, make sense. Just ensure that provided path is absolute.